src/main/java/org/owasp/html/HtmlElementTables.java

```diff
@@ -142,10 +142,10 @@ public HtmlElementTables(
             TD_TAG, new int[] { TR_TAG, TD_TAG, TH_TAG },
             new int[] { TABLE_TAG, TBODY_TAG, TR_TAG }),
         new FreeWrapper(
-            TH_TAG, new int[] { TR_TAG, TD_TAG, TR_TAG },
+            TH_TAG, new int[] { TR_TAG, TD_TAG, TH_TAG },
             new int[] { TABLE_TAG, TBODY_TAG, TR_TAG }),
         new FreeWrapper(
-            TR_TAG, new int[] { TBODY_TAG, THEAD_TAG, TFOOT_TAG, TR_TAG },
+            TR_TAG, new int[] { TBODY_TAG, THEAD_TAG, TFOOT_TAG, TR_TAG, TD_TAG, TH_TAG },
```

Comment:
Why are TD & TH required here?

Reply:
There are three rows in this table.
Without TD, it starts new table from second table row (<TR align='left' bgcolor='#E3E2E2'><TH width='50'>View<TH wi......)
Without TH, it starts new table from third table row (<TR>
<TD height='22' width='50'><A href='http://example.com'>View.......)